src/inmanta/deploy/scheduler.py

```diff
@@ -357,6 +382,43 @@ async def _run_for_agent(self, agent: str) -> None:
 
             self._work.agent_queues.task_done(agent, task)
 
+    async def is_running(self) -> bool:
```

Comment:
this is a bit of a weird name, is doesn't say of the scheduler is running or not, but if, according to the DB, it should be running?